taxi/views.py

```diff
@@ -87,3 +98,40 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form = DriverCreationForm
```

Comment:
In `DriverCreateView`, use `form_class` instead of `form`. It should be `form_class = DriverCreationForm`.